### PR TITLE
Standard One-Way Fares updates

### PR DESCRIPTION
**Pull request by i0sea**
#### Summary of changes
**Asana Ticket:** ["Standard One-Way Fares" updates](https://app.asana.com/0/555089885850811/1202125833856565/f)

create a breakpoint for standard one way fares section on xs small screens
Note: see fares overview link changes have to be made in CMS and should happen before merging this ticket as it removes the current fares link.
Screen Shot 2022-06-07 at 1 37 18 PM

### CODE REVIEW

**Review by zovt — CHANGES_REQUESTED** [review]
Left a couple small comments questions. May also make sense to tag Steve for design feedback as well.

Also a bit confused why we need CMS changes for this? Can we not just stick a link in the header like I did with the user guides PR? (see [1] and [2]. Aside: could maybe make sense to pull that styling up somewhere common)

Let me know if I'm missing anything :)

[1]: https://github.com/mbta/dotcom/pull/1315/files#diff-ed6fd68ea2d7f19efa3b66c08bfe11d39139ff3b0065886af392814a9e1922a7R3
[2]: https://github.com/mbta/dotcom/pull/1315/files#diff-a75ca5591936fdc18c9e64f81708c4fef487efa00f270db254f2c2561f17f1abR228

**Comment by zovt** [comment]
Ah, I see now that the whole block (including the header) is actually coming from the CMS 🤔 interesting. Do we whether or not they are actually able to format the header such that it matches the Abstract design?

File: apps/site/assets/css/_fare-card.scss [hunk 1]
```diff
@@ -1,5 +1,18 @@
+@mixin homepage-fares($homepage:true) {
```
[the remark this answers is not in the document]
Comment (i0sea) [reply apps/site/assets/css/_fare-card.scss:1]: gets called again on line 153

**Comment by i0sea** [comment]
> Ah, I see now that the whole block (including the header) is actually coming from the CMS 🤔 interesting. Do we whether or not they are actually able to format the header such that it matches the Abstract design?

It should. I wrapped the header and link in a div with display: flex and that seems to match. Granted I can't see the changes on dev green since all the styles are wonky there 😅

File: apps/site/assets/css/_fare-card.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (zovt) [reply apps/site/assets/css/_fare-card.scss:1]: ah yep, my eyes totally glazed over the second include there 🙈 thanks!

In that case maybe it makes sense to inline this? something like:

```
<top level>

.c-fares-grid {
  display: none;
}

<under the media breakpoint at line 152>
.c-fare-card {
  display: none;
}
```

might work?

**Review by zovt — APPROVED** [review]
This checks out to me pending the design feedback, thanks for answering my questions :)

**Comment by thecristen** [comment]
@i0sea Can you deploy this to dev-green so we can see the changes along with the header change? BTW I see the header link just fine at the [newer dev-green URL](https://dev-green.mbtace.com) (albeit slightly misaligned).

**Review by lund5000 — CHANGES_REQUESTED** [review]
* "See fares overview" link should have 
  * margin-top: 0.75rem/12px
  * margin-bottom: 1.25rem/20px
* Mode icons should be 1rem/16px square
* "<Mode> One-Way" links should have
  * font-family: Helvetica Neue font
  * margin-bottom: 0.25rem/4px
* "with a" text underneath those links should have
  * font-size: 0.875rem/14px
  * margin-bottom: 0.5rem/8px 
* "Price based" note should have
  * font-size: 0.75rem/12px
  * color: rgb(73, 79, 92)
* .c-fares-grid container should have
  * padding-top: 0.75rem/12px
  * padding-bottom: 1.25rem/20px (doesn't really matter if it's padding on the grid or margin from the last paragraph, though)
* The prices should 
  * have right-aligned text
  * remain padded in by 1rem/16px
  * have a 0.75rem/12px left margin between the text and them 

right-aligned-fares
image

**Comment by i0sea** [comment]
Made requested changes also up on `dev-green`.
Screen Shot 2022-06-08 at 4 10 54 PM

**Comment by lund5000** [comment]
- [ ] "See fares overview" link still needs 12px of space above and 20px below
- [ ] "One-Way" links should have `font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;`
    - Sorry, I was using shorthand before. These need the entire stack used for other headings.
- [ ] The prices should also use `font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;`
- [ ] There should be a total of 20px between the last paragraph in a row and the top border of the next row
- [ ] I think we need to update to  `.m-homepage__fares .c-fares-grid { grid-template-columns: 10% 60% 30%; }`  to give the prices enough space so the dash doesn't take up its own line

image

**Comment by lund5000** [comment]
**TL;DR: I think that the following changes for the xs breakpoint will finish fixing the row height and spacing*
- **removing the padding-bottom value on `div.grid-container`**
- **adding a `div.c-fare-care__note p` that sets `margin-bottom: 0.5rem;`** 

---

There' still too much space at the bottom of each row, which is a combination of
- the padding-bottom value of the containing `div.grid-container`
- the margin-bottom value of `p.c-fare-card__media` or the `<p>` in `div.c-fare-card__note` (which currently have different values)
- the padding-top value of the next `div.grid-container`

It all adds up to a total of about 40-55px instead of the desired 20px total across all those values.

**Comment by i0sea** [comment]
Misunderstood and thought we wanted more padding under the note section. Should be 20px total now. Deploy in progress.
Screen Shot 2022-06-14 at 1 15 42 PM

**Review by lund5000 — APPROVED** [review]
